apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -615,136 +573,60 @@ def from_anndata(
 
                 # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
                 # MS/meas/OBSM,VARM,OBSP,VARP
-                if len(anndata.obsm.keys()) > 0:  # do not create an empty collection
-                    obsm_uri = _util.uri_joinpath(measurement_uri, "obsm")
-                    with _create_or_open_collection(
-                        Collection,
-                        obsm_uri,
-                        ingestion_params=ingestion_params,
-                        context=context,
-                    ) as obsm:
-                        _maybe_set(
-                            measurement, "obsm", obsm, use_relative_uri=use_relative_uri
-                        )
-                        for key in anndata.obsm.keys():
-                            num_cols = anndata.obsm[key].shape[1]
-                            with _create_from_matrix(
-                                # TODO (https://github.com/single-cell-data/TileDB-SOMA/issues/1245):
-                                # consider a use-dense flag at the tiledbsoma.io API
-                                # DenseNDArray,
-                                SparseNDArray,
-                                _util.uri_joinpath(obsm.uri, key),
-                                conversions.to_tiledb_supported_array_type(
-                                    key, anndata.obsm[key]
-                                ),
-                                ingestion_params=ingestion_params,
-                                platform_config=platform_config,
-                                context=context,
-                                axis_0_mapping=jidmaps.obs_axis,
-                                axis_1_mapping=AxisIDMapping.identity(num_cols),
-                            ) as arr:
-                                _maybe_set(
-                                    obsm, key, arr, use_relative_uri=use_relative_uri
-                                )
-                            arr.close()
-                    measurement.obsm.close()
```

Comment:
These `close`s don't exist in the `varm`/`obsp`/`varp` blocks below; they seem unnecessary, but let's confirm or refute that.